src/Chapter3.hs

```diff
@@ -342,6 +342,12 @@ afterwards.
 Define the Book product data type. You can take inspiration from our description
 of a book, but you are not limited only by the book properties we described.
 Create your own book type of your dreams!
+
+data Book = MkBook {
+  pages :: Int
+  , title :: String
+} deriving (Show)
```

Comment:
You can write this data type outside comments to make sure that it actually compiles :heavy_check_mark: